src/main/actions.dsc

```diff
@@ -2,8 +2,15 @@ mortal_actions:
   type: world
   debug: false
   events:
-    after player right clicks block flagged:!mortal.reviving:
+    after player left clicks block flagged:!mortal.gripping|!rclick|!mortal.reviving with:*sword|*axe:
+    - flag player lclick expire:1s
```

Comment:
all flags in the project should be subflags of `mortal`

Reply:
wym